.github/workflows/deploy-godot-editor-release.yaml

```diff
@@ -229,12 +229,12 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: godotengine/godot
-          ref: 3.5.1-stable
+          ref: 4.0-stable
```

Comment:
Shouldn't that be 4.1 ?

Reply:
This PR was created before we were on 4.1.
But now yes.
But i will not be able to fix that until next sunday.
So if any of you two has time, feel free to just change it :-)